.github/workflows/unit-testing.yml

```
@@ -16,7 +16,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: [3.8, 3.12]
+        python-version: [3.8, 3.9, 3.10, 3.11, 3.12]
```

```suggestion
        python-version: ["3.8", "3.9", "3.10", "3.11", "3.12"]
```

(Parece que si no el 3.10 se convierte en 3.1 :)